2014-04-06 Kai Pastor (OpenOrienteering)
- Use correct CMAKE_BIN_SUFFIX for linguist tools when crosscompiling

diff -up Qt5/qttools/src/linguist/linguist.pro.orig Qt5/qttools/src/linguist/linguist.pro
--- Qt5/qttools/src/linguist/linguist.pro.orig	2014-04-06 00:05:36.000000000 +0200
+++ Qt5/qttools/src/linguist/linguist.pro	2014-04-06 09:08:58.679131991 +0200
@@ -10,7 +10,7 @@ qtNomakeTools( \
     linguist \
 )
 
-win32:CMAKE_BIN_SUFFIX = ".exe"
+equals(QMAKE_HOST.os, Windows): CMAKE_BIN_SUFFIX = ".exe"
 
 load(cmake_functions)
 
